libvast/src/system/configuration.cpp

```diff
@@ -176,6 +176,11 @@ caf::error configuration::parse(int argc, char** argv) {
       auto contents = detail::load_contents(config);
       if (!contents)
         return contents.error();
+      // Skip empty config files.
```

Comment:
```suggestion
      // Skip empty config files.
      if (contents->empty())
        continue;
      // Skip config files only containing whitespace.
```

Reply:
> for a 0-byte file, this code path is not triggered

Why would it not be? `all_of` always returns true for empty ranges: https://godbolt.org/z/a5ojGxh93

Reply:
Anyways, pushed a second commit for a separate reason that obsoletes this discussion. See commit message.

Reply:
Makes sense. Green light from my side.